scassandra/src/main/scala/com/evolutiongaming/scassandra/UpdateByIdx.scala

```diff
@@ -0,0 +1,40 @@
+package com.evolutiongaming.scassandra
+
+import com.datastax.driver.core.{GettableByIndexData, SettableData}
+
+trait UpdateByIdx[-A] {
```

Comment:
is there any good reason to use word-token `Idx` over `Index`, like in `GettableByIndexData`?

on other hand, why we do not have `UpdateByNm` instead of `UpdateByName`? ;)

Reply:
This naming pattern already used in the lib thus I prefer to keep code consistent